app/models/post.rb

```diff
@@ -0,0 +1,17 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :comments
+  has_many :likes
+
+  def self.posts_counter(user)
+    where(author: user).count
+  end
+
```

Comment:
- [ ] **[OPTIONAL]** I don't think we would be needing this method after changing the other one to an instance method. Kindly remove it. But if you wish to use it then you should call it correctly in the method since it is a class method.

  ```
  User.posts_counter(user)
  ```
  not
  ```
  posts_counter(user)
  ```